chart/templates/maps-api/deployment.yaml

```diff
@@ -167,15 +167,34 @@ spec:
           {{- if .Values.mapsApi.lifecycleHooks }}
           lifecycle: {{- include "common.tplvalues.render" (dict "value" .Values.mapsApi.lifecycleHooks "context" $) | nindent 12 }}
           {{- end }}
-          {{- if .Values.mapsApi.extraVolumeMounts }}
           volumeMounts:
+            - name: gcp-default-service-account-key
```

Comment:
WTF. Maps-api didn't have access to gcp secrets?????

Reply:
surprise